### PR TITLE
Fix for Capella file import (transposing instruments and grace notes)

### PR DESCRIPTION
**Pull request by AntonioBL**

### CODE REVIEW

**Comment by MarcSabatella** [comment]
What about "regular" grace notes?  Were you also planning on fixing setTpcFromPitch()?

**Comment by AntonioBL** [comment]
Actually, I had a look at that function, but I couldn't find a solution.
I tried to look at how tpc computation is implemented in importove.cpp: tpc is set there with "note->setTpc" on a step2tpc call, but this will set only one tpc value (tpc2 in this case). I tried to use setTpcFromPitch to set tpc1 and then overwrite tpc2 with setTpc called on a step2pitch, but I had the problem that "alteration" seems to be defined differently on cap and capx files. So in the end I decided to rely on setTpcFromPitch and hope that someone will fix it :-)
As you pointed out in the dev-list, by pressing "J" it is possible to cycle through all tpc spelling for that pitch (and key), so it would be a "minor" bug.
So, in conclusion I don't think I will try to fix that function.

**Comment by MarcSabatella** [comment]
Fair enough.  I might take a look at some point.
